external/devlib/devlib/bin/scripts/shutils.in

```diff
@@ -242,26 +242,32 @@ hotplug_online_all() {
 # Misc
```

Comment:
reminder:
that change will need to be removed before merging into next, and somehow replaced by an update of devlib's subtree.